specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json

```diff
@@ -1373,7 +1398,7 @@
       "schema": {
         "$ref": "#/definitions/PatchManagementGroupRequest"
       },
-      "x-ms-parameter-location": "method"
```

Comment:
Same question here - why is the location changed from method signature to client object?